app/views/home/_user_dropdown.html.erb

```diff
@@ -1,5 +1,5 @@
 <li id='notifications_container' class="drawer-pf-trigger dropdown"> </li>
-<li class="dropdown menu_tab_dropdown">
+<li class="dropdown">
```

Comment:
![image](https://user-images.githubusercontent.com/4107560/29861545-e8db2040-8d69-11e7-9c54-67fd210df164.png)

If I click on the notification bell and miss it by a pixel, the user menu opens. I think there should be some "unclickable" space between the two actions.